src/OpenTelemetry.Exporter.Console/ConsoleLogRecordExporter.cs

```diff
@@ -31,6 +36,29 @@ public ConsoleLogRecordExporter(ConsoleExporterOptions options)
 
         public override ExportResult Export(in Batch<LogRecord> batch)
         {
+            if (this.disposed)
+            {
+                if (!this.isDisposeMessageSent)
+                {
+                    lock (this.syncObject)
+                    {
+                        if (this.isDisposeMessageSent)
+                        {
+                            return ExportResult.Failure;
+                        }
+
+                        this.isDisposeMessageSent = true;
+                    }
+
+                    this.WriteLine("The console exporter is still being invoked after it has been disposed. This could be the result of invalid lifecycle management of the OpenTelemetry .NET SDK.");
```

Comment:
> This could be the result of invalid lifecycle management of the OpenTelemetry .NET SDK.

This could be misinterpreted for OTel SDK being at fault. As in the OTel .NET SDK does not do lifecycle management of _something_ correctly. I think we should clearly call out who is at fault.

```suggestion
                    this.WriteLine("The console exporter is still being invoked after it has been disposed. This could be the due to the application's incorrect lifecycle management of the LoggerFactory/OpenTelemetry .NET SDK.");
```

Reply:
updated, thanks